.github/workflows/pytest.yml

```diff
@@ -1,41 +1,50 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: Python package
+name: Pyfive Test
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, modernize_package ]  # leave only "master" after PR69 is merged
   pull_request:
     branches: [ master ]
+  schedule:
+    - cron: '0 0 * * *'  # nightly
 
 jobs:
   build:
 
+    name: Test Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
Is there a compelling reason to drop Python 3.9? Is is supported until the end of Oct 2025 (https://endoflife.date/python)

Reply:
indeed! But a lot of other packages are dropping it like a hot potato - I don't think it's feasible to support it anymore (even if Pyfive has a very small dependency table). Though it's not make or break to me, so I can re-add it, and open an issue to remind us to remove it in eight months :+1:

Reply:
Python 3.9 re-added in https://github.com/jjhelmus/pyfive/pull/69/commits/32158d4d9c937de8d41ba7ef48af040dd89951e3 and https://github.com/jjhelmus/pyfive/pull/69/commits/f2e9bcbbfcd0f9e1c9ea5f98fbe37fd06ac60ed5 :+1: